src/browser/components/AddressBar/AddressBar.js

```diff
@@ -43,6 +44,10 @@ export default class AddressBar extends React.Component {
     );
   }
 
+  handleFocus = (event) => {
+    event.target.select();
```

Comment:
If the user drags to highlight at the same time they're highlighting, will this override the user's selection?  Should we only call `target.select()` if there is no text selected?  That's similar to Chrome's behavior, which would be great to stay in sync with here.